pkg/util/directory_creator.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2023 LY Corporation
```

Comment:
Fix the file name to
directory-creator.go
no `_` unless it is for test like: `directory-creator_test.go`

Reply:
directory generator? as well